### PR TITLE
build: add starter server packages

### PR DESCRIPTION
**Pull request by Anuolu-2020**

### CODE REVIEW

**Comment by Anuolu-2020** [comment]
Good

**Review by Anuolu-2020 — COMMENTED** [review]
Starter Server packages installed

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    // "maxNodeModuleJsDepth": 1,                        /* Specify the maximum folder depth used for checking JavaScript files from 'node_modules'. Only applicable with 'allowJs'. */
+
+    /* Emit */
+    // "declaration": true,                              /* Generate .d.ts files from TypeScript and JavaScript files in your project. */
+    // "declarationMap": true,                           /* Create sourcemaps for d.ts files. */
+    // "emitDeclarationOnly": true,                      /* Only output d.ts files and not JavaScript files. */
+    // "sourceMap": true,                                /* Create source map files for emitted JavaScript files. */
+    // "inlineSourceMap": true,                          /* Include sourcemap files inside the emitted JavaScript. */
+    // "outFile": "./",                                  /* Specify a file that bundles all outputs into one JavaScript file. If 'declaration' is true, also designates a file that bundles all .d.ts output. */
+    "outDir": "./dist" /* Specify an output folder for all emitted files. */,
```
Comment (dev-xero) [inline tsconfig.json:58]: Can we use `build` instead?

Comment (Anuolu-2020) [reply tsconfig.json:58]: Ok will change it

Comment (Anuolu-2020) [reply tsconfig.json]: Change effected u can review it again

**Review by dev-xero — APPROVED** [review]
Alright, looks good 👍🏻

**Pull request merged**